[Den] Week1 월요일 학습 계획 작성

sjuhan123 opened the pull request:
## [개인 프로젝트] 아마존#1주차 월요일 학습계획 작성

- git pr 하는 방식에 대한 학습과 커밋 컨벤션에 대해서 학습
- html 구조 / 태그 / 속성 / 주석에 대해서 학습

sjuhan123 committed:
docs: Week1 day1 학습계획 작성

sjuhan123 committed:
docs: Week1 day1 학습계획 수정 및 추가

crongro reviewed (commented):
HTML 학습부터 하시는 것 좋네요.
이어서 CSS까지 해보세요~

sjuhan123 committed:
docs: Week1 day2 학습계획 추가

sjuhan123 committed:
docs: Week1 day3 학습계획 추가

sjuhan123 committed:
image: 프로젝트 관련 이미지 추가

header icon - basket
header icon - amazon
header icon - flag
header icon - search
header icon - content
header icon svg 파일 내 fill 속성(배경 white로 변경) 설정 완료

sjuhan123 committed:
Feat: 메인 페이지 header 영역 레이아웃 구현

sjuhan123 committed:
feat: css 모듈화를 위한 main 페이지 생성 및 설정

sjuhan123 committed:
feat: 메인 페이지 header 영역 css 파일 생성 및 스타일 설정

sjuhan123 committed:
feat: 메인 페이지 header 영역 modal 오브제 css 파일 생성 및 스타일 설정

sjuhan123 committed:
file: css font 스타일 변수화를 위한 root 파일 생성

sjuhan123 committed:
docs: Week1 day3 까지 아쉬운 점 내용 추가

crongro commented:
커밋로그가 좋네요.

crongro commented on header.css:
```diff
@@ -0,0 +1,230 @@
+.nav-top__inner-wrap {
+  display: flex;
+  flex: 1 1;
+  padding: 8px 0 8px 0;
```
padding : 8px 0; 
해도 동일할거에요.

crongro commented on header.css:
```diff
@@ -0,0 +1,230 @@
+  height: 56px;
+}
+
+.nav-top {
+  display: flex;
+}
```
다른곳에서 display:flex를 사용한다면? 중복이겠네요?

crongro commented on header.css:
```diff
@@ -0,0 +1,230 @@
+
+.nav-top__text{
+  font-weight: 800;
+  font-size: 13px;
```
font-size 설정이 많네요.
기본적으로는 부모노드의 설정값을 상속받는것도 잘 기억해두시고요.
단위가 자주 사용되는 것이라면 css variables로 설정해두는것도 좋겠군요.

crongro commented on main.html:
```diff
@@ -0,0 +1,107 @@
+          </div>
+          <span class="nav-bottom__text">모두</span>
+        </div>
+        <span class="nav-bottom__text">오늘의 딜</span>
+        <span class="nav-bottom__text">고객 서비스</span>
+        <span class="nav-bottom__text">레지스트리</span>
+        <span class="nav-bottom__text">기프트 카드</span>   
+        <span class="nav-bottom__text">판매</span>   
```
네 span을 사용해도 됩니다.
span은 inline 속성의 특징이 있으니 크기를 부여하고 싶다면 div를 써도 되고요.

crongro commented on modal.css:
```diff
@@ -0,0 +1,89 @@
+  flex-grow: 0;
+}
+
+.modal__btn:not(a) {
```
오 요런 selector 도 썼군요.
좋습니다.

crongro reviewed (approved):
css코드 적용을 잘 하셨고요. 다양한 속성들을 이해하고 수정할 수 있도록 학습해보세요.
absolute 로 배치하는것 말고도 다른 방법도 있으니, absolute 를 줄여보는 연습도 해보시고요.
